ancestrymanager/ancestrymanager_test.go

```diff
@@ -1043,3 +1043,114 @@ func TestParseAncestryKey_Fail(t *testing.T) {
 		})
 	}
 }
+
+func TestHandleUnknownProject(t *testing.T) {
+	p := provider.Provider()
+	cases := []struct {
+		name        string
+		data        resources.TerraformResourceData
+		asset       *resources.Asset
+		v1Responses map[string][]*crmv1.Ancestor
+		v3Responses map[string]*crmv3.Project
+		want        []string
+		parent      string
```

Comment:
Could you prefix the things being tested with `want`? for example:

```suggestion
		wantParent      string
```

Reply:
Done.